packages/flutter/lib/src/material/tabs.dart

```diff
@@ -712,7 +712,6 @@ class _TabBarState<T> extends ScrollableState<TabBar<T>> implements TabBarSelect
     setState(() {
       _tabBarSize = tabBarSize;
       _tabWidths = tabWidths;
-      _updateScrollBehavior();
```

Comment:
Good point, it was wrong. The problem with _updateScrollBehavior() was that it reset the scroll position back to zero; not clear why that used to work.